### DISCUSSION TITLE
Nested/child genetic material can be sold

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15146] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

you can sell material inside combined genes

### Reproduction steps

1. get a combined genetic material
2. go to merchant and sell material inside

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
No issues found anymore. Closing discussion as resolved.
